packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -146,7 +146,7 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
         Expanded(
           child: Split(
             axis: splitAxis,
-            initialFirstFraction: splitAxis == Axis.horizontal ? 0.35 : 0.6,
+            initialFirstFraction: 0.35,
```

Comment:
@DaveShuckerow 
What we would really like is for the flex details tab to trigger animating the splitter proportion to a value suitable to display the flex details when the tab is open.